pom.xml

```diff
@@ -73,7 +73,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>display-url-api</artifactId>
-      <version>2.3.1</version>
+      <version>2.3.2-SNAPSHOT</version>
```

Comment:
TODO: Change the version with the incremental one

Reply:
As long as https://github.com/jenkinsci/display-url-api-plugin/pull/26 is merged then https://github.com/jenkinsci/display-url-api-plugin/pull/25 will generated an incremental artifact which could be used in this case